src/components/header/header.tsx

```diff
@@ -0,0 +1,9 @@
+import { StyledHeader } from "./header.styled";
+
+export const Header = () =>
+    <StyledHeader>
+        <h1>conduit</h1>
+        <p>A place to share your knowledge.
```

Comment:
I'd put i18next in place early rather than late - to learn it as well as having a "robust" code base, even if it will only include english.

Reply:
Done